src/Calculator.tsx

```diff
@@ -65,29 +65,29 @@ class Calculator extends React.Component {
         return (
             <div>
               <h1>Confusion Matrix</h1>
-              <table border="1">
+              <table style={{borderWidth: 1}}>
                 <tr>
-                    <td colSpan="2"></td><td colSpan="2" align="center">Reality</td><td>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
+                    <td colSpan={2}></td><td colSpan={2} align="center">Reality</td><td>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
```

Comment:
> src/Calculator.tsx:67:25 - error TS2322: Type 'string' is not assignable to type 'number | undefined'.

You use `{2}` for numbers and `"2"` for strings in JSX, and React wants these to be numbers. Again, it still _works_ if you give it strings, it just doesn't like it.